consensus/common_test.go

```diff
@@ -428,7 +428,7 @@ func randState(nValidators int) (*State, []*validatorStub) {
 
 	vss := make([]*validatorStub, nValidators)
 
-	cs := newState(state, privVals[0], counter.NewApplication(true))
+	cs := newState(state, privVals[0], counter.NewApplication(false))
```

Comment:
Can you explain the rationale for this change? It seems a bit arbitrary.

Reply:
it doesnt do anything for this test. I reverted to avoid polluting the change log